conf/app.routes

```diff
@@ -30,6 +30,10 @@ GET        /saved-declarations                          controllers.SavedDeclara
 
 GET        /saved-declarations/:id                      controllers.SavedDeclarationsController.continueDeclaration(id: String)
 
+GET        /saved-declarations/:id/remove               controllers.RemoveSavedDeclarationsController.displayPage(id: String)
+
+POST       /saved-declarations/:id/remove               controllers.RemoveSavedDeclarationsController.removeDeclaration(id: String)
```

Comment:
DELETE?

Reply:
You can't use DELETE, because forms are in HTML that supports only GET and POST

Reply:
@SteveSugden Most browsers only support POST and GET in HTML forms. Some frameworks RoR have a way of dealing with this. `Play` does not support that.
If this was a RESTful API  then yes `DELETE` should have been the correct choice

Reply:
Good point.  Yes, I was thinking of RESTful APIs